com.chartboost.helium/Android~/build.gradle

```diff
@@ -21,7 +24,7 @@ ext {
     privatePass = onlyPublic? "" : System.getenv("JFROG_PASS")
 
     inputVersionIsHigher = true
-    heliumVersion = '3.1.+'
+    heliumVersion = '3.+'
```

Comment:
this is pretty dangerous to do since when we release 3.2, publishers will unknowingly be upgraded to helium 3.2 whether or not they like it.

Reply:
@bichenwang this is only for safety checks with the Helium Job, basically all it does is check if it compiles with the latest version. Soooo unless you download the updated version, this doesn't affect publishers at all.